.github/workflows/release.yaml

```
@@ -71,7 +76,7 @@ jobs:
           package_root: .rpmpkg
           maintainer: The PRQL Project
           version: ${{ github.ref }}
```

Try this so we can test on non-tags. I'm not confident it'll work.

```suggestion
          version: ${{ github.ref_type == 'tag' && github.refname || 0 }}
```